src/modules/functions.js

```diff
@@ -0,0 +1,11 @@
+import { getMovies } from './API.js';
+import display from './display.js';
+
+const createCards = async () => {
+  // const movies = ['1', '6', '3', '4', '7', '8', '9', '10', '11', '15', '25', '30'];
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.